src/js/administration/components/administrators/__tests__/Create.test.tsx

```diff
@@ -86,7 +86,7 @@ describe("<CreateAdministrator>", () => {
         expect(await screen.findByRole("option", { name: users[0].handle })).toBeInTheDocument();
         expect(await screen.findByRole("option", { name: users[1].handle })).toBeInTheDocument();
 
-        const get_users_scope = mockGetUsersAPI([users[0]], {
+        const get_users_scope = mockFindUsersAPI([users[0]], {
```

Comment:
What does 'scope' mean here?

```suggestion
        const getUsersScope = mockFindUsersAPI([users[0]], {
```

Reply:
`scope` is in reference to the default `Scope` class instance returned by nock when mocking an API route. Honestly not super pumped by the name myself, but the goal was to keep it obvious that the variable is an instance of `Scope` while also specifying which request it is mocking

Reply:
That makes sense.